src/style.css

```
@@ -0,0 +1,31 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(/images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
+
+.stars--1 .stars__star:first-child,
+.stars--2 .stars__star:not(:nth-child(n+3)),
+.stars--3 .stars__star:not(:nth-child(n+4)),
+.stars--4 .stars__star:not(:nth-child(n+5)),
+.stars--5 .stars__star:not(:nth-child(n+6)) {
+  background-image: url(/images/star-active.svg);
+  background-repeat: no-repeat;
+  width: 16px;
+  height: 16px;
+}
```

You already defined these styles in `.stars__star` selector, so there are not needed here
```suggestion
.stars--5 .stars__star:not(:nth-child(n+6)) {
  background-image: url(/images/star-active.svg);
}
```